### PR TITLE
Allow the app pass options to set custom headers for 503s while server is starting

### PR DESCRIPTION
**Pull request by gvspraveen**
During server startup we want to send some header while sending 503. This PR will enable the app to pass their custom headers.

### CODE REVIEW

**Comment by pvenkatakrishnan** [comment]
seems like a straightforward PR to me. :+1:

**Comment by aredridel** [comment]
I'd have called it `startupHeaders` and simplified a bit, but :+1: as it is

**Comment by gvspraveen** [comment]
@aredridel @pvenkatakrishnan  Simplified it to `startupHeaders`

**Comment by aredridel** [comment]
Oh that is delightful. Even less one can have go wrong then!

**Comment by aredridel** [comment]
:+1:

File: index.js [hunk 1]
```diff
@@ -71,6 +71,12 @@ module.exports = function (options) {
         parent.use(function startup(req, res, next) {
             if (promise.isPending()) {
                 res.status(503);
+                var headers = options.startupHeaders;
+                if(headers){
+                    Object.keys(headers).forEach(function(key){
+                        res.header(key, headers[key]);
+                    });
```
Comment (jasisk) [inline index.js:78]: `res.header` can also just [take an object](https://github.com/strongloop/express/blob/dee9fbbbda2f2483c657cf912f68db1a5ba1fcac/lib/response.js#L702-L704). No need to iterate.

**Pull request merged**